cni/nnsgrpcclient/nnsgrpcclient_test.go

```diff
@@ -0,0 +1,117 @@
+package nnsgrpcclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-container-networking/test/nnsmockserver"
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+var mockserver *nnsmockserver.NnsMockServer
+
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
+	teardown()
+	os.Exit(code)
+}
+
+func setup() {
+	mockserver  = nnsmockserver.NewNnsMockServer()
+	go  mockserver.StartGrpcServer(nnsPort)
+	fmt.Printf("mock nns server started\n")
```

Comment:
`fmt.Println` if you don't need formatting directives, also saves you from having to write the newline. Same for below.

Reply:
fixed